e2e/pages/charts/charts.readonly.spec.js

```diff
@@ -3,7 +3,8 @@ import Hooks from '../../utilities/hooks';
 
 const config = require('../../config/mainConfig');
 
-let hooks, page;
+let hooks; let
+  page;
```

Comment:
```suggestion
let hooks; 
let page;
```

I know the auto-fixer does this, but I think it looks terrible...